docs/config.md

```diff
@@ -0,0 +1,56 @@
+# Configuring Perpetual Pools #
+
+Any given instance of a Perpetual Pools deployment consists of various
+parameters that govern various aspects of the system's behaviour. Broadly, these
+are divisible into two categories: factory parameters and pool parameters. The
+latter category consists of the parameters responsible for a given pool and thus
+do not affect the values of the former category.
+
+## Roles ##
+
+These are the *externally-facing* roles in an instance of a Perpetual Pools
+system:
+
+| Name | Description |
+| --- | --- |
+| Fee receiver | TODO |
+| Secondary fee receiver | TODO |
+| Factory owner | TODO |
+| Pool owner | TODO |
+
+## Parameters ##
+
+### Pools ###
+
+These are the configurable parameters of an individual pool:
+
+| Name | Type | Deployment Only? | Who? | Description |
```

Comment:
Think you should specify Deployment only? means that they cannot be modified adhoc post deployment. Might be not obvious to non technical readers.